package-lock.json

```diff
@@ -22,7 +22,7 @@
       "integrity": "sha1-63d99gEXI6OxTopywIBcjoZ0a9I=",
```

Comment:
why all the updates here do youo know?

Reply:
No idea but I'll revert them before merging

Reply:
Reverted